signal/src/lib.rs

```diff
@@ -237,6 +237,7 @@ pub mod pallet {
 		ProposalUnknown,
 		TooManyProposals,
 		TreasuryBalanceLow,
+		TreasuryNotExist,
```

Comment:
Treasury Not Exist stands out from the others in terms of abbreviated english, maybe stick to other pattern or words like undefined, void, invalid?

Reply:
Indeed, it can be similar to `ProposalUnknown` - `TreasuryUnknown`.